extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -81,7 +81,8 @@ iframe.swg-dialog {
   letter-spacing: 0.015em !important;
   font-size: 0.75em !important;
   opacity: 0 !important;
-  transition: opacity 0.3s cubic-bezier(0.42, 0.0, 1, 1) !important;
+  transition: transform 0.3s ease-out !important;
+  margin-right: 0.5em !important;
```

Comment:
Margin will alter layout. If there was another item added to the container it could be effected by this.
Always use the same property that's being animated. In this case it would be `translateX`.

Let's be more subtle with it and use `-0.2em`
The eye evolved to see motion. Even a very small movement will receive attention, especially when it's the only element moving.

This will need to support RTL. in that case it would translate from `0.2em`.